bump.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd $1
+find * -prune -type d | while IFS= read -r d; do
+  echo "attempting to bump $d"
+  cd $d
+  if [ -f requirements.yaml ]; then
+    bump_requirements --user=mesosphere --project=charts --bump-option=sliced --dependencies=*;
```

Comment:
- Is attempting to bump every requirements.yaml the right thing to do?
-  What might trigger this? A cron?